resources/lang/la/navbar.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+return [
+    'Your Profile'  => 'tua Profile',
+    'Your Messages' => 'Nuntius tuus',
+    'Admin'         => 'Maecenas et ipsum',
+    'Logout'        => 'PP',
```

Comment:
@parammittal16  Is "PP" a correct translation? looks weird

Reply:
Yeah you are right its Logout only :)